db/fixtures/miq_product_features.yml

```diff
@@ -4626,11 +4626,11 @@
     - :name: View
       :description: View Providers, Configuration Profiles, Configured Systems
```

Comment:
Should this be updated to "View Configuration Managers, ...", or is it actually showing Providers (in which case we should probably change the identifier)?